script/vary_cache_size_randread.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+#testing different cache sizes and getting the read random results
+
+./db_bench --benchmarks="fillrandom" -num=200000000 -use_direct_reads=true
+:>result_read.log
+
+for size_of_cache in  20000000 200000000 2000000000;
+do
+   ./db_bench --benchmarks="readrandom" -use_existing_db=true -use_direct_reads=True -cache_size ${size_of_cache} -statistics > tmp.log
```

Comment:
using a separate log name for each test to avoid overwriting exsiting logs, instead of using a single name like "tmp.log".
For example: 
```
readrandom_cache_size_${size_of_cache}.log
```